task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         # strip new-line escape symbol from parameter and save it to variable
         imageanddigest=$(echo $imagewithouttag@$IMAGE_DIGEST)
 
-        clair-action report --image-ref=$imageanddigest --db-path=/tmp/matcher.db --format=quay > /tekton/home/clair-result.json || true
+        clair-action report --image-ref=$imageanddigest --db-path=/tmp/matcher.db --format=quay | tee /tekton/home/clair-result.json || true
```

Comment:
is this change intentional?

Reply:
Will create another PR with this change so the name of PR coresponds to the story.